Code/Source/RobotControl/ROS2RobotControlComponent.h

```diff
@@ -21,6 +21,13 @@ namespace ROS2
     {
     public:
         AZ_COMPONENT(ROS2RobotControlComponent, "{CBFB0764-99F9-40EE-9FEE-F5F5A66E59D2}", AZ::Component);
+        ROS2RobotControlComponent()
+        {
+        }
```

Comment:
This can be "=" default'ed
```suggestion
        ROS2RobotControlComponent() = default;
```